src/ConnectKafka.cpp

```diff
@@ -149,3 +157,62 @@ ConnectKafka::GetHighAndLowOffsets(std::string Topic) {
   }
   return HighAndLowOffsets;
 }
+
```

Comment:
On line 152 you can use an initialiser list to create the `OffsetsToSave` struct.